src/app/profile/components/change-password/change-password.component.html

```diff
@@ -1,80 +1,92 @@
-<main>
-	<form-message></form-message>
+<div class="tw-relative tw-w-full">
+	<div class="oeb page-padding">
+		<form-message></form-message>
 
-	<header class="topbar">
-		<div class="l-containerxaxis">
-			<div class="topbar-x-breadcrumbs">
+		<div class="oeb-headline-container-sm">
+			<div class="oeb-breadcrumbs">
 				<bg-breadcrumbs [linkentries]="crumbs"> </bg-breadcrumbs>
 			</div>
 		</div>
-		<div class="l-containerxaxis">
-			<h3 class="topbar-x-heading">Change Password</h3>
-			<p class="topbar-x-subheading">Enter a new password.</p>
+
+		<div class="oeb-headline-container-sm">
+			<h1 hlmH1 class="tw-text-purple">{{ 'Profile.ResetPassword' | translate }}</h1>
+			<p hlmP class="tw-font-normal">{{ 'Profile.EnterNewPassword' | translate }}</p>
 		</div>
-	</header>
-	<div class="l-containerxaxis u-margin-top3x u-width-formsmall">
-		<form class="l-form" [formGroup]="changePasswordForm.rawControl" (ngSubmit)="submitChange()" novalidate>
+
+		<form
+			class="oeb-max-form-width"
+			[formGroup]="changePasswordForm.rawControl"
+			(ngSubmit)="submitChange()"
+			novalidate
+		>
 			<fieldset>
-				<bg-formfield-text
+				<oeb-input
 					[control]="changePasswordForm.rawControlMap.current_password"
-					label="Current Password"
-					errorMessage="Please enter current password"
-					[errorOverride]="!!errors['current_password']"
-					fieldType="password"
+					label="{{ 'Profile.currentPassword' | translate }}"
 					[autofocus]="true"
-				></bg-formfield-text>
-				<p class="u-text-small u-text-error" *ngIf="errors['current_password']">
-					{{ errors['current_password'] }}
-				</p>
+					errorMessage="{{ 'Profile.enterCurrentPassword' | translate }}"
+					[errorOverride]="!!errors['current_password']"
+					[fieldType]="'password'"
+					#issuer_currentPassword_field
+				></oeb-input>
 
-				<div class="u-margin-top2x">
-					<bg-formfield-text
-						[control]="changePasswordForm.rawControlMap.password"
-						label="New Password"
-						[errorMessage]="{
-							required: 'Please enter your new password',
-							minlength: 'Must be at least 8 characters'
-						}"
-						[errorOverride]="!!errors['password']"
-						fieldType="password"
-						sublabel="Must be at least 8 characters"
-					></bg-formfield-text>
-					<p class="u-text-small u-text-error" *ngIf="errors['password']">{{ errors['password'] }}</p>
-				</div>
+				<oeb-input
```

Comment:
![image](https://github.com/user-attachments/assets/bdc6784d-efe4-411a-b9cd-d1dbabd715dc)

There appears a 0 in front of the text if the field has errors.